src/prog_models/utils/containers.py

```diff
@@ -176,6 +190,7 @@ def __repr__(self) -> str:
 
         returns: a string of dictionaries containing all the keys and associated matrix values
         """
-        if len(self.matrix) > 0 and len(self.matrix[0]) == 1:   #the matrix has rows and the first row/list has one value in it
+        if len(self.matrix) > 0 and len(
+                self.matrix[0]) == 1:  # the matrix has rows and the first row/list has one value in it
             return str({key: value[0] for key, value in zip(self._keys, self.matrix)})
-        return str(dict(zip(self._keys, self.matrix)))  
+        return str(dict(zip(self._keys, self.matrix)))
```

Comment:
```suggestion
        return str(dict(zip(self._keys, self.matrix)))

```
Newline at end of file

Reply:
Chris, you suggested the exact same thing that I have there...